### ISSUE TITLE
Can Ekphrasis be used in other languages?

### ISSUE THREAD

**Issue by shuningge** [cbaziotis/ekphrasis#25]
My dataset is in Italian. I am wondering if Ekphrasis can also be used in Italian. Or it's only for English?

**Comment by manueltonneau** [comment]
Yes it can! You basically just have to use the `generate_stats.py` as indicated in the README to train word statistics for your corpus and then use your corpus name when instantiating the different classes.
